crates/ruff/src/registry.rs

```diff
@@ -880,6 +881,18 @@ impl Rule {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct DiagnosticKind {
```

Comment:
As an aside, this might be better-named `DiagnosticMeta` now.

Reply:
I'm not sure on the name `DiagnosticMeta` but to me it seems that this struct is an instantiated (or serialized) diagnostic. In java, you would probably call this a `DiagnosticDto` :D

Reply:
I don't think `DiagnosticMeta` is right either, I almost PR'd that but then I realized it was off too.